rmw_connext_cpp/src/functions.cpp

```diff
@@ -1124,8 +1134,10 @@ rmw_destroy_subscription(rmw_node_t * node, rmw_subscription_t * subscription)
   return RMW_RET_OK;
 }
 
+RMW_LOCAL
```

Comment:
Prevents this symbol being visible in the symbol table of the `.so`. It expands to `__attribute__ ((visibility("hidden")))`. I suppose I could make it `static inline` instead.